pkg/controller/cronjob/cronjob_controllerv2.go

```diff
@@ -555,12 +564,16 @@ func (jm *ControllerV2) syncCronJob(
 	switch {
 	case errors.HasStatusCause(err, corev1.NamespaceTerminatingCause):
 	case errors.IsAlreadyExists(err):
+		// If the job is created by other actor, assume  it has updated the cronjob status accordingly
 		klog.InfoS("Job already exists", "cronjob", klog.KRef(cj.GetNamespace(), cj.GetName()), "job", klog.KRef(jobReq.GetNamespace(), jobReq.GetName()))
+		return cj, nil, err
 	case err != nil:
 		// default error handling
 		jm.recorder.Eventf(cj, corev1.EventTypeWarning, "FailedCreate", "Error creating job: %v", err)
 		return cj, nil, err
 	}
+
+	metrics.CronjobCreationSkew.Observe(time.Since(*scheduledTime).Seconds())
```

Comment:
It might be better to measure from the scheduled time to the creationTimestamp of the job object.  That way the metric matches exactly what is observed by users.

Reply:
@dashpole I am not sure. The thing that trips me is, if the user wants to look at creation timestamp, why is it obvious for the user to look at the job `creationTimestamp` and not at the pod/s that is running the actual code for the job?

Reply:
ahh well, the description you suggested makes it clear. if the user wants to see the metrics for pods, they should be looking at jobs metrics, not cronjob metrics, +1 I think I answered my own questions, will update, +1

Reply:
Yeah, in general, I just want to be able to easily tell what the metric is measuring from the description, and make sure that it _actually is_ measuring what you say it is measuring. :)